src/base/TilesRendererBase.js

```diff
@@ -32,13 +34,18 @@ export class TilesRendererBase {
 
 	}
 
-	constructor( url ) {
+	//The url can be:
+	//A plain url to the tileset json file
+	//A Cesium Ion asset number with access token
+	//A url to the Cesium Ion json file with bearer token as the ionAccessToken. This needs to be fetched from the Ion endpoint separately
+	constructor( url, ionAccessToken = defaultIonToken ) {
```

Comment:
It would naturally be better to load assets via an options object instead. This is mostly a quick example.